messaging/src/main/java/org/axonframework/common/caching/JCacheAdapter.java

```diff
@@ -66,6 +67,21 @@ public boolean putIfAbsent(Object key, Object value) {
         return jCache.putIfAbsent(key, value);
     }
 
+    @Override
+    public <T> T computeIfAbsent(Object key, Supplier<T> valueSupplier) {
```

Comment:
Looks like this method can be omitted to use the default implementation of the interface.